src/app/shared/support/support.html

```diff
@@ -0,0 +1,48 @@
+<div class="docs-help-support">
+  <ul>
+    <li>
+      <a href="https://stackoverflow.com/questions/tagged/angular-material">
+        <img src="../../../assets/img/homepage/stackoverflow-logo.svg" alt="StackOverflow icon">
+        StackOverflow
+      </a>
+      <div class="docs-help-support-description">Questions tagged with <a
+        href="https://stackoverflow.com/questions/tagged/angular-material">angular-material</a> or
+        <a href="https://stackoverflow.com/questions/tagged/angular-cdk">angular-cdk</a></div>
+    </li>
+    <li>
+      <a href="https://gitter.im/angular/material2">
```

Comment:
We now recommend discord instead of gitter. I'd suggest replacing the link.

Reply:
i replaced Discord with Gitter!